app.py

```diff
@@ -0,0 +1,55 @@
+from flask import Flask, render_template, request, redirect, url_for, abort, flash 
+from model.models import db, Todo, StatusEnum
+from flask_migrate import Migrate
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///todos.db' 
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+app.secret_key = 'your_secret_key'
+db.init_app(app)
+migrate = Migrate(app, db)
+
+
+@app.route("/", methods=["GET", "POST"])
+@app.route("/home", methods=["GET", "POST"])
+def home():
+    if request.method == "POST":
+        todo_name = request.form["todo_name"]
+        description = request.form.get("description", "")
+        if not todo_name:  
+            flash("Please enter a task name.", "error")
+            return redirect(url_for("home"))
+        new_todo = Todo(name=todo_name , description=description)
+        if new_todo.checked == StatusEnum.COMPLETED:
```

Comment:
When a task is created, its status is set to pending. Why is this logic being used here? Also, write the database logic in utils.

Reply:
@pythexcel sir because if any new task is created in default it will be in pending while we check it becomes completed